src/Logikfabrik.Umbraco.Jet/Logikfabrik.Umbraco.Jet.nuspec

```diff
@@ -15,7 +15,7 @@
     <copyright>Copyright (c) 2016</copyright>
     <tags>Umbraco, Jet, MVC, Code First, uJet</tags>
     <dependencies>
-      <dependency id="UmbracoCms.Core" version="[7.5.6,8.0]" />
+      <dependency id="UmbracoCms.Core" version="[7.6.0,8.0]" />
```

Comment:
The project files reference UmbracoCms.Core 7.6.2. Shouldn't the NuGet dependency min version be 7.6.2 too? I'm not that up-to-date with the minor version changes.